src/library_html5.js

```diff
@@ -284,7 +284,7 @@ var LibraryJSEvents = {
 
   _findEventTarget__deps: ['_maybeCStringToJsString', '_specialEventTargets'],
   _findEventTarget: function(target) {
-    var domElement = __specialEventTargets[target] || document.querySelector(__maybeCStringToJsString(target));
+    var domElement = __specialEventTargets[target] || (typeof document !== 'undefined' ? document.querySelector(__maybeCStringToJsString(target)) : undefined);
```

Comment:
We should avoid the extra code size here (as shown in the test number updates) if it isn't needed. Please use
```
#if ENVIRONMENT_MAY_BE_WORKER
#endif
```
around the new added code.

Reply:
Oh OK. There's similar code through the library whenever `document` is queried. I guess I'll add an ifdef around those. I'm not really familiar with node, can I assume that document is defined in that case? Should the ifdef be `#if ENVIRONMENT_MAY_BE_WORKER || ENVIRONMENT_MAY_BE_NODE`?

Reply:
I don't think node has a DOM environment, so there is no `document` etc.